src/components/todoform/TodoForm.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { UserShape } from '../shapes/UserShape';
+import './TodoForm.scss';
+
+export class TodoForm extends React.PureComponent {
+  state = {
+    title: '',
+    userName: '',
+    titleError: false,
+    userNameError: false,
+  }
+
+  handleChange = (event) => {
+    const { name, value, type } = event.target;
+
+    this.setState({
+      [name]: type === 'text'
+        ? value.replace(/[^\w ]+/, '')
+        : value,
+      [`${name}Error`]: false,
+    });
+  }
```

Comment:
nicee

Reply:
Yura doesn't agree😄 
let's create an 'errors' field in the state and put there all the errors info, so then you can change each error value like this 
`erros[errorName]: false,`